### PR TITLE
fix list/get tool api

### PR DESCRIPTION
**Pull request by ylwu-amzn**
### Description
[Describe what this change achieves]
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: ml-algorithms/src/main/java/org/opensearch/ml/engine/tools/PainlessScriptTool.java [hunk 1]
```diff
@@ -30,8 +30,9 @@ public class PainlessScriptTool implements Tool {
 
     @Setter @Getter
     private String alias;
+    private static String DEFAULT_DESCRIPTION = "User this tool to get index information.";
```
Comment (HenryL27) [inline ml-algorithms/src/main/java/org/opensearch/ml/engine/tools/PainlessScriptTool.java:33]: nit: "Use this tool to run a Painless script" (or something. does the agent learn about the tools from the descriptions?)

File: ml-algorithms/src/main/java/org/opensearch/ml/engine/tools/VectorDBTool.java [hunk 2]
```diff
@@ -44,8 +44,9 @@ public class VectorDBTool implements Tool {
     public static final String NAME = "VectorDBTool";
     @Setter @Getter
     private String alias;
+    private static String DEFAULT_DESCRIPTION = "Useful for when need to search my data in OpenSearch index.";
```
Comment (HenryL27) [inline ml-algorithms/src/main/java/org/opensearch/ml/engine/tools/VectorDBTool.java:47]: if description does go to the LLM, then probably want to have consistent language.
"Use to search over data in my OpenSearch index."
Does OpenSearch even need to be mentioned here?

File: ml-algorithms/src/main/java/org/opensearch/ml/engine/tools/PainlessScriptTool.java (hunk 1 above)
Comment (ylwu-amzn) [reply ml-algorithms/src/main/java/org/opensearch/ml/engine/tools/PainlessScriptTool.java:33]: yes

File: ml-algorithms/src/main/java/org/opensearch/ml/engine/tools/VectorDBTool.java (hunk 2 above)
Comment (ylwu-amzn) [reply ml-algorithms/src/main/java/org/opensearch/ml/engine/tools/VectorDBTool.java:47]: Good point. Will fine tune all of these later

**Review by dhrubo-os — APPROVED** [review]

**Pull request merged**